src/platform/linux/x11/mod.rs

```diff
@@ -290,9 +290,10 @@ impl EventsLoop {
                     let written = unsafe {
                         use std::str;
 
+                        const BUFF_SIZE: usize = 512;
                         let mut windows = self.windows.lock().unwrap();
                         let window_data = windows.get_mut(&WindowId(xwindow)).unwrap();
-                        let mut buffer: [u8; 16] = [mem::uninitialized(); 16];
+                        let mut buffer: [u8; BUFF_SIZE] = [mem::uninitialized(); BUFF_SIZE];
```

Comment:
If 16 isn't guaranteed to be large enough, how do we know that 512 is? Is there a way to dynamically allocate overflow space if needed?